server/util/query_builder/query_builder_test.go

```diff
@@ -1,13 +1,44 @@
 package query_builder_test
 
 import (
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/buildbuddy-io/buildbuddy/server/util/query_builder"
 	"github.com/stretchr/testify/assert"
 )
 
+// normalize applies a few simple normalization to make query assertions easier.
+func normalize(query string) string {
+	// The current logic in this func assumes there are no strings in order for
+	// the meaning to be equivalent, since it strips whitespace without
+	// considering whether the spaces are inside strings or not.
+	if strings.Contains(query, "\"") || strings.Contains(query, "'") || strings.Contains(query, "`") {
+		panic("normalizeSpace cannot yet handle string or backtick literals")
```

Comment:
instead of panic. can we do t.Error? or return an error. and use require.NoError at the caller site?

Reply:
Done